test/WebSites/MvcTagHelpersWebSite/Views/MvcTagHelper_Home/Script.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@addtaghelper "*, Microsoft.AspNet.Mvc.TagHelpers"
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>Script</title>
+    <script src="~/blank.js" data-foo="foo-data1">
```

Comment:
To promote good use, we should probably throw all these script tags at the bottom of the body :smile: